src/app/components/ContentBanner/TextItem.jsx

```diff
@@ -67,6 +74,15 @@ const TextItem = ({
               {content.desc ? <p>{content.desc}</p> : null}
             </div>
         }
+        <button
```

Comment:
Can the button live inside its own div so that it can be positioned relative to  <a> and .hpContentBanner-content-description?

Reply:
It could, but it doesn't seem to make a difference.